### PR TITLE
Added SSJ Feature files 

### PR DESCRIPTION
**Pull request by lfcabrera14**

### CODE REVIEW

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/OWM_Vacancy_Manager.feature [hunk 1]
```diff
@@ -279,4 +279,65 @@ Feature: OWM Vacancy Manager Scenarios
...
+APPTRACK-576
+APPTRACK-578
+APPTRACK-577
+APPTRACK-91
+APPTRACK-93
+APPTRACK-94
+APPTRACK-440
+APPTRACK-562
+APPTRACK-443
+APPTRACK-88
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/OWM_Vacancy_Manager.feature:342]: What is the purpose of Story IDs commented?

**Review by hamidnesar — COMMENTED** [review]
What is the purpose of commented Story IDs across all feature files?

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/Applicant Profile.feature [hunk 2]
```diff
@@ -43,11 +43,45 @@ Feature: Applicant Profile Scenarios
       | Jessica   | Marianna   | Mollick   | mollickja@nih.gov  | 7018211343 | 2123323454    | Masters          | Yes             | 11 Kolls | 6         | Sausalito | CA    | USA     | 97712 |
       | Karen     | Elena      | Andrew    | andrewkl@nih.gov   | 8233212333 | 2202122234    | Bachelors        | Yes             | 11 Bells | 39        | Sacramento| CA    | USA     | 95512 |
 
-  @alenan @SSJ-712 @Bug_Open @juarezds @In_Progress @playwright
+  @juarezds @SSJ-32 @SSJ-33 @SSJ-36 @SSJ-92 @SSJ-98 @Regression @playwright @Bug_Open
```
[the remark this answers is not in the document]
Comment (lfcabrera14) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/Applicant Profile.feature:46]: yes there is still a bug open for those tests, also I added the test case IDs to each feature as a TODO list for automating those tests

**Review by hamidnesar — APPROVED** [review]
Reviewed and approved!

**Review by iamdez99 — APPROVED** [review]
Reviewed and approved - juarezds

**Pull request merged**